src/main/scala/com/mwz/sonar/scala/sensor/ScalaSensor.scala

```diff
@@ -69,6 +69,16 @@ final class ScalaSensor extends Sensor {
     }
   }
 
+  def ExampleIssues: Unit = {
+    val isEmpty = List(1).size == 0
+    val b = if (true) true else false
+    var TWO = 1 + 1
```

Comment:
MINOR: Var could be val ([more](http://localhost/coding_rules?open=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.VarCouldBeVal&rule_key=sonar-scala-scapegoat%3Acom.sksamuel.scapegoat.inspections.unneccesary.VarCouldBeVal))